helm/templates/common/ingress.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.common.ingress.enabled }}
-apiVersion: networking.k8s.io/v1beta1
+apiVersion: networking.k8s.io/v1
```

Comment:
`networking.k8s.io/v1` is only available in K8S 1.19+ and GKE is still on 1.17/1.18 (stable/regular channel). That's why I had reverted back to `v1beta1` in
https://github.com/flyteorg/flyte/pull/916/commits/f35a37486ec2026d550d0f253c2b330684a17a53

Reply:
aah ok

Reply:
it seems they were supposed to release 1.19 April / 18. seems slipping?

Reply:
Oh you're right [1.19 is GA in GKE](https://cloud.google.com/kubernetes-engine/docs/release-notes#119_ga) since two weeks.